<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowOpacity">
   <double>0.960000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Administrator Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Mono L</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suspicious 
Human Activity 
Detection System</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitLogin">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>20</width>
     <height>471</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
